scrapers/scrape_bl.py

```diff
@@ -125,6 +126,21 @@
                             rows[row_date]['cases'] = int(c[1] or 0) + int(c[2] or 0) + int(c[3] or 0)
                         rows[row_date]['recovered'] = c[1]
                         rows[row_date]['deaths'] = c[3]
+            else:
+                data = sc.find(r'<pre id="data_1".*?> ?Datum,&quot;Normale Station&quot;,&quot;Intensivstation \(nicht beatmet\)&quot;,&quot;Intensivstation \(beatmet\)&quot;\s*([^<]+)</pre>', d)
+                if data:
```

Comment:
instead of `if data` and the error message in the `else`, I'd use `assert data, "bug in regex"`

Reply:
good idea ! -> changed